src/index.html

```diff
@@ -1,17 +1,158 @@
+        </label>
+
+        <label for="agreement" class = "form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+        <label class = "form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class = "form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="9"
+            maxlength="15"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <label for="cats-yes">
+            Yes
+            <input
+            type="radio"
+            name="loveCats"
+            id="cats-yes"
+            value="yes"
+            checked
+          >
+          </label>
+
+          <label for="cats-no">
+          No
+            <input
+            type="radio"
+            name="loveCats"
+            id="cats-no"
+            value="no"
+          >
+          </label>
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="timeToGoToBed">
+        </label>
+
+        <label class="form-field">
+          What is your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rangeScore"
+            min="0"
+            max="10"
+            step="2"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Additional info</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            cols="15"
+            rows="2"
+            autocomplete="off"
+          ></textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit</button>
+    </form>
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
в кінці коду завжди має бути пустий рядок